examples/param_est.ipynb

```diff
@@ -0,0 +1,234 @@
+   "metadata": {},
+   "source": [
+    "For this example, we will define specific parameters that we want to estimate.\n",
+    "\n",
+    "We can pass the desired parameters to our __keys__ keyword argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "keys = ['thrower_height', 'throwing_speed']"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To really see what `estimate_params()` is doing, we will print out the state before executing the estimation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Printing state before\n",
+    "print('Model configuration before')\n",
+    "for key in keys:\n",
+    "    print(\"-\", key, m.parameters[key])\n",
+    "print(' Error: ', m.calc_error(times, inputs, outputs, dt=1e-4))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we will run `estimate_params()` with the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.estimate_params([(times, inputs, outputs)], keys, dt=0.01)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's see what the new parameters are after estimation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('\\nOptimized configuration')\n",
+    "for key in keys:\n",
+    "    print(\"-\", key, m.parameters[key])\n",
+    "print(' Error: ', m.calc_error(times, inputs, outputs, dt=1e-4))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sure enough- parameter estimation determined that the thrower's height wasn't 20 m, instead was closer to 1.9m, a much more reasonable height!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`estimate_params()` creates a structure of 'runs' by constructing each index of times, inputs, and outputs and placing them into a tuple.\n",
+    "\n",
+    "                `runs = [(t, u, z) for t, u, z in zip(times, inputs, outputs)]`\n",
+    "\n",
+    "\n",
+    "Using our optimization function, which runs `calc_error()` as a subroutine (more information about `calc_error()` found in our Calculating Error Example), given each of the runs.\n",
+    "\n",
+    "You can also adjust the metric that is used to estimate parameters by setting the error_method to a different `calc_error()` method.\n",
+    "e.g., m.estimate_params([(times, inputs, outputs)], keys, dt=0.01, error_method='MAX_E')\n",
+    "Default is Mean Squared Error (MSE)\n",
+    "See calc_error method for list of options."
```

Comment:
This is a really nice additional note. Should we include a line of code here to illustrate it? (basically pull out line 206 and make it an executable statement?)